### PR TITLE
Added link to infosec.mozilla.org under the security guide.

### PR DESCRIPTION
**Pull request by atombrella**
### Summary

I think this guide merits to be widely known. It contains a condensed list of various HTTP flags, settings for cookie (I suppose `SameSite`, `httpOnly` and `secure` ought be mentioned under the cookie section as well, in addition to details to the fact that cookies are encrypted), and a link to their SSL/TLS configuration tool. 

### Other Information

These three sites are linked somehow from the above site.
https://observatory.mozilla.org
https://securityheaders.com
https://www.ssllabs.com/

### CODE REVIEW

**Comment by atombrella** [comment]
It's worth noting that https://observatory.mozilla.org rates https://rubyonrails.org/ poorly. In particular, the HSTS header is not implemented.

File: guides/source/security.md [hunk 1]
```diff
@@ -1215,4 +1215,5 @@ The security landscape shifts and it is important to keep up to date, because mi
 * Subscribe to the Rails security [mailing list](https://groups.google.com/forum/#!forum/rubyonrails-security).
 * [Brakeman - Rails Security Scanner](https://brakemanscanner.org/) - To perform static security analysis for Rails applications.
 * [Keep up to date on the other application layers](http://secunia.com/) (they have a weekly newsletter, too).
+* Mozilla has [web security guidelines](https://infosec.mozilla.org/guidelines/web_security.html) that cover various HTTP headers, cookies and TLS configuration.
```
Comment (vipulnsward) [inline guides/source/security.md:1218]:
```suggestion
* [Mozilla's Web Security Guidelines](https://infosec.mozilla.org/guidelines/web_security.html) - Recommendations on topics covering Content Security Policy, HTTP headers, Cookies, TLS configuration, etc.
```

Comment (atombrella) [reply guides/source/security.md:1218]: :+1:

Comment (atombrella) [reply guides/source/security.md]: The Secunia link right above my change is actually kind of broke. I think it should probably be https://www.flexera.com/resources.html?type=blog ?

Comment (vipulnsward) [reply guides/source/security.md]: Yeah I was going to remove it separately. You can go ahead and remove in this one, it points to some products now, not super useful.

Comment (atombrella) [reply guides/source/security.md]: Done :+1: I think the Infosec documents are quite good in many ways.

**Comment by vipulnsward** [comment]
Thanks! I forgot to mention, please add a [ci skip] in commit message in future doc-only changes for us to avoid the CI runs.

**Pull request merged**